python/nammu/view/EditSettingsView.py

```diff
@@ -273,22 +413,30 @@ def save(self, event=None):
         working_dir = self.wd_field.getText()
 
         # Read the fontsize from the textfield
-        fontsize = self.fs_field.getText()
+        console_fontsize = self.fs_field.getText()
+        edit_area_fontsize = self.edit_area_fs_field.getText()
 
-        # Use isnumeric() to test if a unicode string only has digits
-        if fontsize.isnumeric() and (8 <= int(fontsize) <= 30):
-            pass
-        else:
-            self.logger.error("Invalid font size. Please enter a number "
-                              "between 8 and 36.\n\n"
-                              "Font size set to default value: 14")
-            fontsize = 14
+        # Validate the working directory input string
+        working_dir = self.validate_working_dir(working_dir)
+
+        # Validate the input fontsizes
+        console_fontsize = self.valiate_fontsize(console_fontsize, 'console')
+        edit_area_fontsize = self.valiate_fontsize(edit_area_fontsize,
```

Comment:
Same typo as earlier.

Reply:
And you could be passing the config key directly instead of "console".

Reply:
Typo fixed in ca65983. Regarding the passing of the console key, see my comment above.